src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -844,4 +844,22 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_PackageDetailsUrlLink" xml:space="preserve">
     <value>Package details link</value>
   </data>
+  <data name="Label_Deprecated" xml:space="preserve">
+    <value>Deprecated</value>
+  </data>
+  <data name="Label_OrAbove" xml:space="preserve">
+    <value> (or above)</value>
+  </data>
+  <data name="Label_DeprecationReasons_CriticalBugs" xml:space="preserve">
+    <value>This package has been deprecated because it has critical bugs.</value>
+  </data>
+  <data name="Label_DeprecationReasons_Legacy" xml:space="preserve">
+    <value>This package has been deprecated because it's legacy and no longer maintained.</value>
+  </data>
+  <data name="Label_DeprecationReasons_LegacyAndCriticalBugs" xml:space="preserve">
+    <value>This package has been deprecated because it's legacy and has critical bugs.</value>
```

Comment:
"Legacy" is a bit of an abstract word to me as the implications aren't clear. I would prefer "no longer maintained" instead:

```suggestion
    <value>This package has been deprecated due to critical bugs and is no longer maintained.</value>
```

Reply:
Same here. What about gallery? The unmodified text is currently shipping as part of gallery. https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Views/Packages/_DisplayPackageDeprecation.cshtml